schema/common/defaults/configuration/logging.yml

```diff
@@ -13,12 +16,20 @@ specification:
   - readall
   - snapshotrestore
   paths:
-    data: /var/lib/elasticsearch
-    repo: /var/lib/elasticsearch-snapshots
-    logs: /var/log/elasticsearch
+    opensearch_home: /usr/share/opensearch
```

Comment:
I would prefer simpler structure, for example:
```
paths:
  opensearch:
    home: /usr/share/opensearch
    conf_dir: /usr/share/opensearch/config
    log_dir: /var/log/opensearch
    [...]
  opensearch_dashboards:
    home: /usr/share/opensearch-dashboards
    conf_dir: /usr/share/opensearch-dashboards/config
    plugin_bin: /usr/share/opensearch-dashboards/bin/opensearch-dashboards-plugin
```

Reply:
My proposal is to move this remark to the separate issue in order to not block this PR as this is seems not to be a crucial change.